docs/introduction/getting-help.md

```diff
@@ -1,6 +1,3 @@
-:construction: We're still working on ehrQL and its documentation:
-**hearing what you find tricky to do or understand will help us improve ehrQL**.
-
```

Comment:
I think this comment still provides value, in that it contextualises the rest of this page.

Reply:
I disagree. The context of this page is how the user can get help from us; this comment is about how we can get help from the user.